controllers/list/submissions/PKPSubmissionsListHandler.inc.php

```diff
@@ -160,6 +160,7 @@ public function getConfig() {
 			'selectAllLabel' => __('common.selectAll'),
 			'viewMore' => __('list.viewMore'),
 			'viewLess' => __('list.viewLess'),
+			'updated' => __('common.updated'),
```

Comment:
I _think_ this is no longer used, is it?

Reply:
right...